### ISSUE TITLE
Accessibility evaluation of timeliner - human review of timeliner show page

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#6435]
# Description

Timeliner is commonly used, but is a visualization tool. How can this be usable for someone with low vision or without vision?

Although Dananji has tested with keyboard navigation and automated accessibility testers, we haven't done human-based testing against the WCAG success criteria or screen reader testing.

Test with screenreader to see if it makes any sense. Can we embed accessibility for this use case into the current version of timeliner of do we need a specific alternate version of timelines?

Color in timelines is also an issue. Faculty often use color to create patterns of meaning in a timeline. We should have some kind of indicator that pairs with color that is accessible to a screenreader to allow for these patterns to be interpreted (ex: red color has a label that says "red").

## Done Looks Like
- [ ] Team discussion/Swarm to review Timeliner, how it works with assistive technologies currently, and what can be improved
- [ ] How much effort is required to add these to Timeliner?

**Comment by joncameron** [comment]
Possible that not all accessibility checker tools are not seeing all of this in Avalon because iframes may be unavailable or hidden to these tools. We should test against both the Timeliner demo site (Timeliner by itself) and embedded within Avalon. How the DOM renders things may be different in the different contexts.

Timeliner is a particular concern because of its use in pedagogical settings at IU and because the nature of the project means it had not been developed with accessibility in mind.

**Comment by joncameron** [comment]
@dananji will create initial tickets based on this investigation.
